### PR TITLE
P2 201 fe payroll history

### PR DESCRIPTION
**Pull request by ALFAROSO**
https://team-voyagers.atlassian.net/jira/software/projects/P2/boards/2?assignee=712020%3Aa4d78f0d-920a-427a-a5ce-b36979797950&selectedIssue=P2-201

### CODE REVIEW

**Review by ReinhardtBotha — APPROVED** [review]
💵

File: src/pages/Payroll/Payroll.tsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+import { useState } from 'react';
+import { Box, Button } from "@mui/material";
+import { DataGrid, GridColDef } from "@mui/x-data-grid";
+import { FormattedMessage, useIntl } from "react-intl";
+import BasicHome from "layouts/BasicHome";
+import useQueryCache from 'hooks/useQueryCache';
+import { useQuery } from 'react-query';
+import { getPayrollHistory } from 'api/payroll';
+  
+  const CustomPayroll = () => {
```
Comment (whamester) [inline src/pages/Payroll/Payroll.tsx:11]: rename component to Payroll

file name and component name must be the same

File: src/pages/Payroll/Payroll.tsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+
+  const formatDate = (timestamp : number) => {
+    const date = new Date(timestamp);
+    const month = date.toLocaleString("default", { month: "short" });
+    const day = date.getDate();
+    return `${month} ${day}`;
+  };
+
+  const columns : GridColDef[] = [
```
Comment (whamester) [inline src/pages/Payroll/Payroll.tsx:35]: move the columns out of the component

File: src/pages/Payroll/Payroll.tsx [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+      onSuccess: (results) => {
+        setPayrollData(results);
+      },
+      onError: (error) => {
+        console.log(error);
+      },
+    });
+
+
+  const formatDate = (timestamp : number) => {
```
Comment (whamester) [inline src/pages/Payroll/Payroll.tsx:28]: note: refactor this format date to use intl messages

**Review by whamester — CHANGES_REQUESTED** [review]

**Review by whamester — APPROVED** [review]

**Comment by whamester** [comment]
Pending: format date with intl

**Review by BryanCaldeira — APPROVED** [review]

**Pull request merged**